.readthedocs.yml

```diff
@@ -0,0 +1,5 @@
+python:
```

Comment:
Is this enough ? How is this going to know that it needs to build it beforehand ?

Reply:
pip knows per pyproject.toml 👍

Reply:
ah install method is pip.